compose.yml

```diff
@@ -0,0 +1,28 @@
+services:
+  webui:
+    build: .
+    volumes:
+      - type: bind
+        source: ./models
+        target: /webui/models
+      - type: bind
+        source: ./outputs
+        target: /webui/outputs
+      - type: bind
+        source: ./extensions
+        target: /webui/extensions
+      - type: bind
+        source: ./embeddings
+        target: /webui/embeddings
+      - type: bind
+        source: ./configs
```

Comment:
there're already several yaml files under `configs` folder, did you mean user can customized their own config to override the default ones ,

Reply:
From what I've gathered that's where the customizations made in the webui are persisted